account.go

```diff
@@ -0,0 +1,22 @@
+package chartmogul
+
+const (
+	accountEndpoint = "account"
+)
+
+// Account details in ChartMogul
+type Account struct {
+	Name        string `json:"name,omitempty"`
```

Comment:
omitempty matters only during serialization, ie. if you had a `POST` method.

Reply:
I took a look into the documentation:

The "omitempty" option specifies that the field should be omitted **from the encoding** if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string. - [https://golang.org/pkg/encoding/json/](https://golang.org/pkg/encoding/json/)

```go
// Field appears in JSON as key "myName" and
// the field is omitted from the object if its value is empty
Field string `json:"name,omitempty"`
```
Since we are not creating, adding or updating the data, I agree, I will remove it.